src/MergePlugin.php

```diff
@@ -388,6 +388,9 @@ public function onPostInstallOrUpdate(ScriptEvent $event)
                     '</error>'
                 );
                 file_put_contents($lock, $lockBackup);
+                // propagate error to the parent process to allow proper status code being returned
+                // this is important for any CI/CD tools to properly recognize if an error occurred in this phase
+                exit(100);
```

Comment:
Why 100? Wouldn't `1` be sufficient? Also, we could add the flag that enables this feature for BC

Reply:
Lower number status codes have a special though unofficial meaning. I didn't want to collide with them. It is not a big deal, having unique status code helps pinpoint the potential error and lower the chances that somewhere in the code it will be reused.

Reply:
We have also the status code from the command above we could also use that?